package.json

```diff
@@ -10,8 +10,8 @@
     "test": "cross-env CI=true lerna run test --since origin/master -- --coverage",
     "create-plugin": "backstage-cli create-plugin",
     "lint": "lerna run lint",
-    "storybook": "yarn workspace @spotify-backstage/core storybook",
-    "build-storybook": "yarn workspace @spotify-backstage/core build-storybook"
+    "storybook": "yarn workspace storybook start",
```

Comment:
Just submitted #380 to fix this 🙈 , also got rid of the top-level build command though, for reasons mentioned in the PR. Should be alright to just run the build-storybook script inside the storybook package?

Reply:
it's alright. 👍🏻
so, i'll update the build script in config.